hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSConfigKeys.java

```diff
@@ -1557,6 +1557,17 @@
   public static final double
       DFS_DATANODE_RESERVE_FOR_ARCHIVE_DEFAULT_PERCENTAGE_DEFAULT = 0.0;
 
+
+  public static final String
+      DFS_NAMESERVICES_RESOLUTION_ENABLED =
```

Comment:
Will these configs used by other services (JournalNode, ZKFC)? If not, can we specify the datanode in the prefix?

Reply:
Yeah, I was thinking we can expand it for other components to use, so leave it without datanode prefix.

Reply:
Sure, I will add datanode prefix